gui/src/components/processes/ProcessListTable.svelte

```diff
@@ -45,131 +51,197 @@
   }
 </script>
 
-<table>
-  <thead>
-    <th />
-    <th>Process</th>
-    <th>Logs</th>
-    <th />
-  </thead>
-  {#each data as { id, name, running } (id)}
-    <tr>
-      <td>
-        <ProcessRunControls {setProcRun} {statusPending} {id} {running} />
-      </td>
-
-      <td
-        ><a
-          class="process-name"
-          use:link
-          href={`/workspaces/${encodeURIComponent(
-            workspaceId,
-          )}/processes/${encodeURIComponent(id)}`}>{name}</a
-        ></td
+{#each data as { id, name, running } (id)}
+  <div class="card" style={logStyleFromHash(name + ':out')}>
+    <div>
+      <ProcessRunControls {setProcRun} {statusPending} {id} {running} />
+    </div>
+
+    <div>
```

Comment:
Would be good to extract these cards into their own component files. Smaller components are usually better.

Reply:
Yeah, this is part of the mentioned cleanup :) thanks for the review